Library/Homebrew/PATH.rb

```diff
@@ -10,13 +10,10 @@ class PATH
 
   delegate each: :@paths
 
-  # FIXME: Enable cop again when https://github.com/sorbet/sorbet/issues/3532 is fixed.
-  # rubocop:disable Style/MutableConstant
```

Comment:
I think these comments still need to stay around but perhaps in a different format otherwise I worry they will get "lost" when changing `Style/MutableConstant` back. @issyl0 can you do this in a follow-up PR? Thanks!

Reply:
Given this `T.type_alias` thing is unfixable on our side, it might be worth creating a cop that injects into `Style/MutableConstant` and allows `T.type_alias`.

Something similar to how https://github.com/Shopify/rubocop-sorbet/blob/main/lib/rubocop/cop/sorbet/mutable_constant_sorbet_aware_behaviour.rb works for `T.let`. Unlike `T.type_alias`, `T.let` can be frozen, just not in a way RuboCop understands. What we'd do differently for `T.type_alias` is we'd probably do a `return if t_type_alias` in `on_assignment` rather than passthrough.